ansible/roles/debops.postfix/defaults/main.yml

```diff
@@ -256,6 +271,351 @@ postfix__tls_dh512_param_file: '{{ ansible_local.dhparam[postfix__dhparam_set]
                                        ansible_local.dhparam[postfix__dhparam_set]|d())
                                    else "" }}'
                                                                    # ]]]
+# .. envvar:: postfix__no_log [[[
+#
+# This variable can be used in the Ansible tasks to set the ``no_log``
+# parameter. Changing it to ``False`` can help with debugging issues with the
+# roles, but otherwise it should be set to ``True`` to ensure that Ansible
+# doesn't leak sensitive data in its logs.
+postfix__no_log: True
+                                                                   # ]]]
+                                                                   # ]]]
+# Virtual Mail [[[
+# ----------------
+#
+# This settings help to configure to enable `postfix` to host multiple (virtual) domains,
+# and thus provide email to several domains with just one `mail server`.
+# Currently the Virtual Mail support works only with LDAP enabled, in the future `mariaDB` could be enabled.
+#
+
+# .. envvar:: postfix__ldap_enabled [[[
+#
+# In order to enable Virtual Mail support, you need to enable LDAP authentication support manually.
+# It is disabled by default in order to define a single Mail Relay Server.
+# When enabled, `postfix` will configure mail-boxes locally and allow LDAP authenticated users to
+# send email over this instance.
+postfix__ldap_enabled: 'False'
+                                                                   # ]]]
+# .. envvar:: postfix__virtual_mail_posix_user [[[
+#
+# Virtual Mail POSIX username
+# For the accesses to the mailbox directories a separate user `vmail` (`Virtual Mail`) is created,
+# under which the accesses of Postfix, Dovecot and other components of the mail server should take place.
+# On the one hand this prevents mail server components from accessing sensitive system directories,
+# on the other hand it protects the mailboxes from external access.
+# Only vmail (and root) are allowed to access the mailboxes.
+postfix__virtual_mail_posix_user: 'vmail'
+                                                                   # ]]]
```

Comment:
I'm not sure how close Postfix Virtual Mail is tied to LDAP/MariaDB/PostgreSQL support. Perhaps the `vmail` setup could be moved into its own separate role, for example `debops.postfix_virtual`, that depends on `debops.postfix` to configure Postfix itself.

Virtual mail, especially the UNIX account part, would probably be useful for other roles as well, like `debops.dovecot`. You could expose information about the `vmail` account via Ansible local facts, and `debops.dovecot` role could use them to configure the mail backend.